libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -445,11 +445,26 @@ void AP_RangeFinder_LightWareI2C::update(void)
 
 void AP_RangeFinder_LightWareI2C::sf20_timer(void)
 {
+#ifdef MFG_TEST_BUILD
+    static const uint32_t TIMER_FREQ = 20;
```

Comment:
recommend adding units to this FREQ variable

Reply:
Done!